server/sonar-web/src/main/js/apps/overview/stickers/StickerSnippet.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+import ClipboardButton from '../../../components/controls/ClipboardButton';
+
+interface Props {
+  snippet: string;
+}
+
+export default function StickerSnippet({ snippet }: Props) {
+  return (
+    <div className="sticker-snippet">
+      <pre>{snippet}</pre>
```

Comment:
the command doesn't always fit:

![image](https://user-images.githubusercontent.com/2317341/34830836-6b155348-f6e5-11e7-8f81-0381f8984acb.png)

I think it's better to forbid wrapping and display a horizontal scrollbar.